app/javascript/components/notification-drawer/notification-drawer.jsx

```diff
@@ -0,0 +1,192 @@
+                            </MenuItem>
+                            <MenuItem
+                              id={`dropdownRemove-${notification.id}`}
+                              bsClass="dropdown"
+                              eventKey="3"
+                              header={false}
+                              onClick={
+                                () => dispatch(clearNotification(notification, Boolean(maxNotifications)))
+                              }
+                            >
+                              {__('Remove')}
+                            </MenuItem>
+                          </Drawer.Dropdown>
+                          <Icon
+                            className="pull-left"
+                            name={getNotficationStatusIconName(notification)}
+                            type="pf"
+                            onClick={
+                              () => notification.unread && dispatch(markNotificationRead(notification.id))
+                            }
+                          />
+                          <NotificationContent>
+                            <NotificationMessage
+                              title={notification.message}
+                              onClick={
+                                () => notification.unread && dispatch(markNotificationRead(notification.id))
+                              }
+                            >
+                              {notification.message}
+                            </NotificationMessage>
+                            <NotificationInfo
+                              leftText={moment(notification.timeStamp).format('MM/DD/YYYY')}
+                              rightText={moment(notification.timeStamp).format('h:mm:ss A')}
+                            />
+                          </NotificationContent>
+                        </Notification>
+                      ))}
+                      { maxNotificationsConstant < totalNotificationsCount && (
+                        <Notification id="notificationLimitBar" key="notificationLimit" className="text-center" seen>
+                          <NotificationContent>
+                            <NotificationMessage
+                              title="title"
+                            >
+                              { maxNotifications ? __(`Showing ${maxNotifications} items out of ${totalNotificationsCount} total.`) : __(`Showing all ${notifications.length} items.`) }
+                            </NotificationMessage>
+                            <a
+                              id="toggleMaxNotifications"
+                              className="btn btn-link"
+                              onClick={() =>
+                                dispatch(toggleMaxNotifications())
+                              }
+                            >
+                              { maxNotifications ? __('Show all (may take a while)') : __(`Show only the first ${maxNotificationsConstant}`)}
```

Comment:
This is not going to work. Get text translation (the __() function) does not work with es6 template strings. You should use `sprintf` if you want to use variables in translations.
```jsx
__(`Show only the first ${maxNotificationsConstant}`) => sprintf(__(Show only the first %s'), maxNotificationsConstant)
```